### ISSUE TITLE
Setup guide for windows

### ISSUE THREAD

**Issue by LaCulotte** [barronwaffles/dwc_network_server_emulator#564]
Hi,

Thanks to the discord user **waffles**, I found this great installation guide for windows : [https://github.com/sepalani/MH3SP/wiki/Setup](https://github.com/sepalani/MH3SP/wiki/Setup)
I think having a link to this tutorial in the README.md or in the wiki could be a good idea, it would have saved me many hours of search.
(Also this issue is there to share the link to anyone who would stumble on this ;) ) 

Thanks !

**Comment by sepalani** [comment]
@LaCulotte 
Could you please be more precise regarding the issue you encountered? Which section/parts of the wiki worked or wasn't working? Just so you know, the setup used in the MH3SP isn't using a (Apache) reverse proxy which means some HTTP servers won't be working properly. MH3 isn't using these servers, that's why I advised to use the alternative config (which isn't ideal in many situations).

When I'll manage to find some more time, I'll try to address some of these issues, including but not limited to: the wiki, python3 support _(while trying to preserve python2 support for capable but legacy systems)_, automation _(docker/docker-compose?)_, etc.

**Comment by LaCulotte** [comment]
The issue I encontered is that there is no indication on where to begin, except for the requirements in the README.md. While I figured that the script to launch was master_server.py, and I still felt clueless. 
It might be of personnal preference, but having a setup example on a setup working shows me the entry point and gives me more information on what I am doing right/wrong when I troubleshoot. Moreover, the guide gives a little info about how it the emulator works and this can also help with troubleshooting.
The point of this issue is more to share the info I found in the a way I can, because I think it could help.

That being said, if you feel that this info might be missleading or if it does not belong here, I can delete the issue.

**Comment by ElHave247** [comment]
Yeah is sad developers made than vague instructions and linux only
